.taskcluster.yml

```diff
@@ -29,6 +29,14 @@ tasks:
         then: ${event.after}
         else: ${event.release.tag_name}
 
+    push_date_time:
+      $if: 'tasks_for == "github-pull-request"'
+      then: ${event.pull_request.head.repo.pushed_at}
```

Comment:
I *think* this is when the repository was last updated, not this pull request. I suspect that `event.pull_request.updated_at` is likely to be more relevant than this. I suspect the github-push event is similar.